test/Microsoft.ML.Tests/CopyColumnEstimatorTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.ML.Tests
+{
+    public class CopyColumnEstimatorTests
```

Comment:
>CopyColumnEstimatorTests [](start = 17, length = 24)

The transform behavior is checked in `TestDataPipeBase.TestCore`. So we need to only check the estimator behavior: that we throw at schema errors for example.

I expect we need to create an estimator workout test as well, similar to `TestCore`. Not in this PR though.